### PR TITLE
Doc 184

### PR DESCRIPTION
**Pull request by prabaadorsys**
updated the script which is trigger codecov command based on the pr branch.

### CODE REVIEW

File: .travis/codecov_bash.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```
Comment (valb3r) [inline .travis/codecov_bash.sh]: How it is used?

File: .travis.yml [hunk 2]
```diff
@@ -36,5 +35,6 @@ before_install:
   - sudo mv ./openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /usr/local/bin
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash) -f '!*datasafe-business*' -F unittests
-  - bash <(curl -s https://codecov.io/bash) -f '*datasafe-business*' -F e2e_tests
\ No newline at end of file
+  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```
Comment (valb3r) [inline .travis.yml]: This should be bash script

**Comment by prabaadorsys** [comment]
testing

File: .travis.yml (hunk 2 above)
Comment (prabaadorsys) [reply .travis.yml]: updated and added as bash script file.

**Pull request merged**